ct_ansi.go

```diff
@@ -4,10 +4,14 @@ package ct
 
 import (
 	"fmt"
+	"os"
 	"strconv"
 )
 
 func resetColor() {
+	if os.Getenv("TERM") == "dumb" {
```

Comment:
Question, do you expect the value of `os.Getenv("TERM")` to change during the execution of a process?

If not, consider factoring it out into a package level variable, e.g.:

```Go
var dumbTerminal = os.Getenv("TERM") == "dumb"
```

Then use it like this:

```Go
if dumbTerminal {
    return
}
```